### DISCUSSION TITLE
Rename the project

### DISCUSSION THREAD

**Discussion by roee88** [fybrik/fybrik#608] [Ideas] [answered]
Hi all,

I'm gathering proposals for a new name for the project.
Preferably a catchy and non descriptive name. It would also be good for the name to find very few hits in google. 

Please:
1. Add a suggestion as an answer to this discussion (not as a comment to an answer).
2. Vote up for existing suggestions that you like. We will count thumbs up.

**Comment by roee88** [comment]
**guberno**

Meaning:
1. to pilot, steer a ship
2. to govern, manage

Latin; from a non-Indo-European Mediterranean language; Akin to Ancient Greek κυβερνάω (kubernáō) which is the verb for Kubernetes.

**Comment by ronenkat** [comment]
`Dedomena 
`
The word "data" translated into Greek

**Reply by lga-zurich** [reply]
Just for the record, this has a really bad ring in Spanish, if hat matters.

**Comment by ronenkat** [comment]
Open Data Use Controller
Acronym:
1. ODU-C (122,000 results)
2. OpenDUC (2,960 results)

**Comment by tomersolomon1** [comment]
`Dasmos` 
Meaning duty, customs, tax, etc.

Intuition: Just as the customs officials look at cargo you import, our project enforces policies on imported data.

`Foros` is also a nice name (with the same meaning), but maybe has too many search results..

**Comment by michaelfactor** [comment]
L8ttice -- a lattice is a mesh, 8 as a reference to k8s and coming close to the pronunciation of the word lattice.  There are essentially no google matches so very unique

**Comment by lga-zurich** [comment]
Following the Greek Kubernetes theme, [Astynomikos or astynómos](https://en.wiktionary.org/wiki/%CE%B1%CF%83%CF%84%CF%85%CE%BD%CE%BF%CE%BC%CE%B9%CE%BA%CF%8C%CF%82) is the "policeman" in Greek, so the policy enforcer.

I would call the project `Astynomic`; it has a good "autonomic" ring to it.

**Comment by andreagreggo2** [comment]
I like the greek theme idea @lga-zurich! 

What about calling it "demok8tia"  or some variation? 
Based on demokratia, or “rule by the people” (from demos, “the people,” and kratos, or “power”). I guess in this instance our power is Kubernetes :) I think the power of the Mesh for Data is that it frees data to deliver value for all, hence it democratizes enterprise data.

**Comment by chris-giblin** [comment]
How about something around the concepts of "mediation" or "medium" ? M4D mediates access. Keeping with the Greek theme, then "mesitévo". 

I also thought of "Mediterranean" as it derives from "medius" (middle), has a warm blue association, and  on its shores M4D was conceived :). Of course "Mediterranean" is a long and very common word, breaking the distinctness constraint. Nevertheless I throw it out there since it may lead to another idea.

**Comment by rohithdv** [comment]
`beluga` :- based on the oceanic dolphin (commonly called white whale) in consistent with the theme of sea based terms and also indicative of secure movement of data ? 

another could be from Harry potter theme (not sure about copyright violations) `gringotts` which is the bank for wizards from which it is impossible to steal anything (data in our context).

**Reply by roee88** [reply]
The problem is that there are too many search results for these words.

**Comment by roee88** [comment]
`fybrik`

As a component in a data fabric solution.

**Comment by roee88** [comment]
Shomair / Shomeyr

by @simanadler

**Reply by simanadler** [reply]
Means to guard in Hebrew ... i.e. we're guarding the data.
Could be spelled in other ways ... shomeyr?

**Comment by ronenkat** [comment]
We have two running candidate which entered into the final evaluation phase.
- fybrik
- Astynomic

**Comment by ronenkat** [comment, marked as answer]
We have settled on **fybrik** as the new project name.
